xbmc/ServiceManager.h

```diff
@@ -153,6 +157,7 @@ class CServiceManager
   std::unique_ptr<CPlatform> m_Platform;
   std::unique_ptr<PLAYLIST::CPlayListPlayer> m_playlistPlayer;
   std::unique_ptr<CSettings> m_settings;
+  std::unique_ptr<KODI::GAME::CControllerManager> m_controllerManager;
```

Comment:
I actually moved this from `CGameServices` because input (and peripherals) depend on some controller stuff, but game services are brought up last. There still exists `CGameServices::GetController*()`, but this is only called from within GAME.